docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+metrics gatherer still continues to gather the alerts (not in JSON) as well, but this can be removed in the future,
+and we will keep the data only in JSON.
+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/config/alerts.json](./insights-archive-sample/config/alerts.json)
 
-* Location in archive: config/alerts.json
-* See: docs/insights-archive-sample/config/alerts.json
-* Id in config: active_alerts
-* Since version:
-  - 4.12+
+### Location in archive
+- `config/alerts.json`
+
+### Config ID
+`clusterconfig/active_alerts`
+
+### Released version
+- 4.12.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## CRD
 
-collects the specified Custom Resource Definitions.
+Collects the specified Custom Resource Definitions.
 
 The following CRDs are gathered:
-- volumesnapshots.snapshot.storage.k8s.io (10745 bytes)
-- volumesnapshotcontents.snapshot.storage.k8s.io (13149 bytes)
+- `volumesnapshots.snapshot.storage.k8s.io` (10745 bytes)
+- `volumesnapshotcontents.snapshot.storage.k8s.io` (13149 bytes)
+
+### API Reference
+None
 
-The CRD sizes above are in the raw (uncompressed) state.
+### Sample data
+- [docs/insights-archive-sample/config/crd](./insights-archive-sample/config/crd)
 
-* Location in archive: config/crd/
-* Id in config: clusterconfig/crds
+### Location in archive
+- `config/crd/{name}.json`
+
+### Config ID
+`clusterconfig/crds`
+
+### Released version
+- 4.6.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## CephCluster
 
 Collects statuses of the`cephclusters.ceph.rook.io` resources
 from Openshift Data Foundation Stack.
 
-### API Reference:
-https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
+### API Reference
+- https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
 
-### Sample data:
-docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json
+### Sample data
+- [docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json](./insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json)
 
-### Location in archive:
-| Version | Path													 |
-| ------- | ----------------------------------------------------- |
-| <= 4.12 | config/storage/{namespace}/{name}.json 				 |
-| 4.13+   | config/storage/{namespace}/cephclusters/{name}.json 	 |
+### Location in archive
+| Version   | Path													 |
+| --------- | ------------------------------------------------------- |
+| < 4.13.0  | config/storage/{namespace}/{name}.json 				 |
+| >= 4.13.0 | config/storage/{namespace}/cephclusters/{name}.json 	 |
 
-### Config ID:
-clusterconfig/ceph_cluster
+### Config ID
+`clusterconfig/ceph_cluster`
 
-### Since versions:
-* 4.8.49
-* 4.9.48
-* 4.10.31
-* 4.11.2
-* 4.12
+### Released version
+- 4.12.0
+
+### Backported versions
+- 4.8.49+
+- 4.9.48+
+- 4.10.31+
+- 4.11.2+
+
+### Changes
+None
```

Comment:
I would like to include include a warning about the file collisions with OpenshiftStorage/StorageCluster in OCP 4.11 and OCP 4.12. Not sure if the "Changes" section is the best place. It works for me.

Something like this would do (please check specifically that the links are correct):

```suggestion
* In OCP 4.11 and OCP 4.12, the location of gathered data collides with data gathered by the [GatherStorageCluster](#storagecluster) gatherer. It is practically impossible to tell the two resources apart. Use with caution.
```